app/2.0/docs/devguide/dom-template.md

```diff
@@ -10,7 +10,7 @@ By default, adding a DOM template to an element causes Polymer to create a shado
 
 The DOM template is also processed to enable data binding and declarative event handlers.
 
-## DOM template overview
+## Specify a DOM template
 
 To specify an element's DOM template:
 
```

Comment:
(3) is useful when elements are not being defined via HTML imports, e.g. when using ES modules.

(2) may be useful in the module case, or also to extend/modify super templates, share templates, etc.